x/profile/keeper/msg_update_validator_description.go

```diff
@@ -40,6 +40,9 @@ func (k msgServer) UpdateValidatorDescription(
 	}
 
 	k.SetValidator(ctx, validator)
+	err := ctx.EventManager().EmitTypedEvent(
```

Comment:
The message follows the same logic as `MsgAddValidatorOperatorAddress`.If the validator didn't exist, it is created and therefore should be a `ValidatorCreated` event